flask_github.py

```diff
@@ -24,6 +24,45 @@
 null_handler = logging.NullHandler()
 _logger.addHandler(null_handler)
 
+#: MIME type of JSON
+JSON_MIMETYPE = 'application/json'
+
+#: Generic (default) MIME type if response headers are not set
+GENERIC_MIMETYPE = 'application/octet-stream'
```

Comment:
Can you remove these 2 constants please? It makes no sense to me to make these strings constant other than requiring a jump while reading the code.